src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{age=" + age + ", name='" + name + '\''
+                + ", ownerName='" + ownerName + '\'' + '}';
```

Comment:
I think it would be better to simplify the code for reading in this way:
```
        return "Cat{"
                + "age=" + age
                + ", name='" + name + '\''
                + ", ownerName='" + ownerName + '\''
                + '}';
```

Reply:
I'm agree.